docs/self-hosted/install/install-on-kubernetes.md

```diff
@@ -4,45 +4,46 @@ url: /docs/self-hosted/latest/install/install-on-kubernetes/
 
 # Install Gitpod Self-Hosted on Kubernetes
 
-This section describes how to install Gitpod on any Kubernetes cluster.
-Gitpod also provides more optimized installations offering better performance for particular cloud providers:
-* *Google Cloud Platform*: Install Gitpod in a blank GCP project [using a script that automates the procedure](../install-on-gcp-script/).
-* *AWS*: Install Gitpod on AWS [using a script that automates the procedure](../install-on-aws-script/).
+This section describes how to install Gitpod on any Kubernetes cluster using [Helm](https://helm.sh). This ist the most flexible and generic way of installing Gitpod.  
 
-Gitpod is installed using [Helm](https://helm.sh).
-The source of the chart can be found [here](https://github.com/gitpod-io/gitpod/blob/master/chart/).
```

Comment:
The reference to the helm chart is there to guide users to the code and enable them to start discussing and contributing. I'd vote for leaving it here, somewhere.

Reply:
good point. Re-added to the first paragraph.